rust/kcl-wasm-lib/Cargo.toml

```diff
@@ -24,7 +24,8 @@ data-encoding = "2.9.0"
 futures = "0.3.31"
 # Enable the feature in a transitive dependency.
 getrandom_2 = { package = "getrandom", version = "0.2", features = ["js"] }
-getrandom_3 = { package = "getrandom", version = "0.3", features = ["wasm_js"] }
+# Enable wasm_js feature for all transitive dependencies using getrandom 0.3
+getrandom = { version = "0.3", features = ["wasm_js"] }
```

Comment:
@jtran I'm wondering if this will help with my windows build issues, will test in the morning